src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Infrastructure/Internal/ResolverUIMessageRouterCommunicator.cs

```diff
@@ -11,23 +11,21 @@
  * or implied. See the License for the specific language governing permissions
  * and limitations under the License.
  */
-
-using System;
+ 
 using System.Text.Json;
 using Finos.Fdc3;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Contracts;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Converters;
-using MorganStanley.ComposeUI.Messaging;
-using MorganStanley.ComposeUI.Messaging.Abstractions;
+using MorganStanley.ComposeUI.MessagingAdapter.Abstractions;
 
 namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Infrastructure.Internal;
 
 internal class ResolverUIMessageRouterCommunicator : IResolverUICommunicator
```

Comment:
I think this class should be renamed to avaoid messagerouter namings (also its field should be renamed.)